go/bind/keybase.go

```diff
@@ -214,6 +214,8 @@ func Init(homeDir, mobileSharedHome, logFile, runModeStr string,
 		EK:      config.GetEKLogFile(),
 	}
 
+	fmt.Printf("Go: Using config: %+v\n", kbCtx.Env.GetLogFileConfig(config.GetLogFile()))
```

Comment:
rm?

Reply:
it only shows up in the xcode console as far as i know, figured it was worth leaving in case someone was wondering what happened to other log files logged in as a test user

Reply:
(we log the logfile paths above that)